src/ptf/packet.py

```diff
@@ -42,6 +42,20 @@
 BOOTP = scapy.layers.dhcp.BOOTP
 PADDING = scapy.packet.Padding
 
+BTH = None
+if not config.get("disable_rocev2", False):
+    try:
+        ptf.disable_logging()
+        import scapy.contrib
+        import scapy.contrib.roce
+        BTH = scapy.contrib.roce.BTH
```

Comment:
I'm curious about why we don't use `scapy.main.load_contrib` like we do for the other cases?

Reply:
We use now. Thanks for pointing this out.